bin/play

```diff
@@ -6,16 +6,18 @@ $: << File.join(File.dirname(__FILE__), "..", "lib")
 
 require "display"
 require "board"
-require "players"
+require "player_module"
```

Comment:
module is redundant as that is what it 'is' but not what it does.  player_factory would be better.

Reply:
Thanks this is really helpful. I'll rename it.